content/tutorials/aws-persistence.textile

```diff
@@ -109,7 +111,7 @@ From the "AWS dashboard":http://console.aws.amazon.com/:
 
 h2(#create-lambda). Create the Lambda function
 
-We can now create the Lambda role itself. Lambda functions as the logic center and pipeline for persistence in this tutorial, you can set up more comprehensive functions than we will here to optimize the persistence to suite your business needs.
+You can now create the Lambda role itself. Lambda is used as the logic center and pipeline for persistence in this tutorial. Although you will be doing a straightforward function you can set up more comprehensive functions to suit your business needs.
```

Comment:
Maybe make the purpose of the Lambda clearer, and avoid the use of the word "pipeline":

_Lambda functions can be as complex as you need them to be. The Lambda function you will create in this tutorial is very simple: it will process incoming messages and persist them to a text file._